### DISCUSSION TITLE
Opt-in instead of Opt-out

### DISCUSSION THREAD

**Discussion by fyr77** [snarfed/bridgy-fed#833] [Ideas]
Hi, I think this is a really cool project. However, I do believe that having it opt-out is not a good way of handling this, especially since it provides posts to a completely different service that is run by a single corporation.
In essence, it's similar to a bot scraping public fedi posts - possible, but certainly not intended or ethical. 

Perhaps instead a sign-up for instances could be done? One option that comes to mind here is to implement it as a kind of relay: instance admins can subscribe to it, which opts it in.

Thank you for your consideration.

**Comment by snarfed** [comment]
I hear you! A number of us in the fediverse moderation community discussed and thought this through. The resulting opt-out decision wasn't unanimous, of course, but it definitely wasn't made on a whim. Lots of background in https://snarfed.org/2024-01-19_moderate-people-not-code ; I'd love to hear thoughts on that if you have any.

The relay idea is interesting! I only know a bit about them, I'll definitely go learn more and consider it.

**Reply by rocketdyke** [reply]
"a number of us"

were any of that number from marginalized communities? somehow I doubt it.

**Comment by temtemy** [comment]
Also requiring users to opt-out on launch is probably a bad idea as we don't know yet if there will be serious interop issues or not. Perhaps you may be confident that won't happen, but we have not seen that yet. The fediverse is more than Mastodon, and the chances it's gonna be janky in say Misskey is non-zero.

Please make it opt-in for at least a few months after BS starts federating.
